index.html

```diff
@@ -1,476 +1,534 @@
+                            will test the relationship.</p>
+                    </div>
+
+                </div>
+                <div id="cs5"onclick="deletehtml(this.id)" class="content_small">
+
+                    <img src="img/tank.jpg" class="img_small">
+                    <div>
+                        <h2>`Shared resentments ' but Russia ties could be 'awkward 'for China</h2>
+                        <p> Russia and china's leaders have met more than 30 times since 2013 ,but the Ukraine crises
+                            will test the relationship.</p>
+                    </div>
+
+                </div>
+                <div id="cs6"onclick="deletehtml(this.id)" class="content_small">
+
+                    <img src="img/cover1.jpg" class="img_small">
+                    <div>
+                        <h2>`Shared resentments ' but Russia ties could be 'awkward 'for China</h2>
+                        <p> Russia and china's leaders have met more than 30 times since 2013 ,but the Ukraine crises
+                            will test the relationship.</p>
+                    </div>
+
+                </div>
+            </div>
+        </div>
+    </div>
+    <section class="large">
+        <img src="img/20220224_2_52306047_73986646.jpg" class="large_img">
+        <p class="title_large">Ukraine-Russia Crisses</p>
+        <div class="slider_container">
+            <div class="slider_box">
+                <img src="img/cover4.jpg" class="imagesmall">
+                <p> Rejection in the diplomacey of bitcoin its a fake new .</p>
+            </div>
+            <div class="slider_box">
+                <img src="img/cover3.jpg" class="imagesmall">
+                <p> Rejection in the diplomacey of bitcoin its a fake new .</p>
+            </div>
+            <div class="slider_box">
+                <img src="img/cover2.jpg" class="imagesmall">
+                <p> Rejection in the diplomacey of bitcoin its a fake new .</p>
+            </div>
+        </div>
+    </section>
+    <div class="related_topic">
+        <div>
+            <p>MOST READ </p>
+            <div class="square"></div>
+        </div>
+        <div class="related_topic--text">
+            <p> Ukraine is now responaine on the russian attacks.</p>
+            <p> Ukraine is now responaine on the russian attacks.</p>
+            <p> Ukraine is now responaine on the russian attacks.</p>
+            <p> Ukraine is now responaine on the russian attacks.</p>
+            <p> Ukraine is now responaine on the russian attacks.</p>
+            <p> Ukraine is now responaine on the russian attacks.</p>
+            <p> Ukraine is now responaine on the russian attacks.</p>
+            <p> Ukraine is now responaine on the russian attacks.</p>
+            <p> Ukraine is now responaine on the russian attacks.</p>
+        </div>
+    </div>
+    <div class="news--small">

```

Comment:
not following BEM

Reply:
isn't  small  considered as a  modifier  in here ? or i should make it news separated and small as a separated class